docs/docs/06-For CISOs/Security/01-principles.md

```diff
@@ -50,3 +50,16 @@ Defense in depth is a security strategy that employs multiple layers of defense
 ## Redundancy
 
 Redundancy is a key principle in metal-stack's security architecture. It involves duplicating critical components and services to ensure that if one fails, others can take over, maintaining system availability and reliability. This is particularly important for data storage and processing, where redundancy helps prevent data loss and ensures continuous operation.
+
+## BMC User Management 
+
+For bare metal provisioning with metal-stack, two dedicated users interacting with a machine BMC are created.
+The `metal-hammer` first creates a BMC user called `root` or `superuser` with the administrator privilege. The password used, is configured with the Ansible variable [`metal_api_bmc_superuser_pwd`](https://github.com/metal-stack/metal-roles/blob/master/control-plane/roles/metal/README.md#metal-api). It is necessary e.g. for [`metal-bmc`](https://github.com/metal-stack/metal-roles/blob/master/partition/roles/metal-bmc/README.md), to perform its actions while deleting a machine and adding it to the pool of available machines again.
```

Comment:
Theoretically, this feature is optional. If you do not pass the super user password, this functionality gets disabled. But then metal-bmc just cannot report data for this machine when it is not allocated.